config/crd/patches/cainjection_in_initialbuildpipelineselectors.yaml

```diff
@@ -0,0 +1,7 @@
+# The following patch adds a directive for certmanager to inject CA into the CRD
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    cert-manager.io/inject-ca-from: $(CERTIFICATE_NAMESPACE)/$(CERTIFICATE_NAME)
```

Comment:
includes dependency on cert-manager which is not currently installed by infra-deploymets. Also the patch is not referenced.

Reply:
It's auto-generated by operator-sdk. Will clean up later, thanks.